core/src/test/java/feign/client/AbstractClientTest.java

```diff
@@ -146,6 +146,22 @@ public void parsesErrorResponseBody() {
     }
   }
 
+  @Test
+  public void parsesUnauthorizedResponseBody() {
+    String expectedResponseBody = "ARGHH";
+
+    server.enqueue(new MockResponse().setResponseCode(401).setBody("ARGHH"));
+
+    TestInterface api = newBuilder()
+        .target(TestInterface.class, "http://localhost:" + server.getPort());
+
+    try {
+      api.postForString("HELLO");
+    } catch (FeignException e) {
+      assertThat(e.contentUTF8()).isEqualTo(expectedResponseBody);
```

Comment:
Is there anything we can check on the server?

Reply:
I don't think so, it responses as normal just fine